tscat_gui/edit.py

```diff
@@ -169,7 +169,7 @@ def validate(self, word: str, pos: int) -> QtGui.QValidator.State:
 class CustomAttributesGroupBox(AttributesGroupBox):
 
     def create_label(self, text: str):
-        attrs = list(self.entity.variable_attributes_as_dict().keys())
+        attrs = list(self.entity.variable_attributes_as_dict().keys()) + self.entity._fixed_keys
```

Comment:
This really looks like a code smell to me, why using a private member?

Reply:
Because there is no method for in tscat (ftm). I used _fixed_keys in several places already. I shall create a bug/feature request on tscat's side.

Reply:
yes, I prefer since this is likely a trivial PR.